src/main/scala/firrtl/ir/IR.scala

```diff
@@ -8,6 +8,7 @@ import Utils.indent
 /** Intermediate Representation */
 abstract class FirrtlNode {
   def serialize: String
+  override def toString = serialize
```

Comment:
Mmmm I don't think we should do this... ToString is useful to see the in-memory representation, as opposed to the concrete serialization.